### PR TITLE
Adding windows event log integration

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

We do want a new integration as the two existing one (Event Viewer and Win_32_Logs) might disapear in the future and we do not want to confuse user between Events and Logs.

### Motivation

PM request

### Preview link

* https://docs-staging.datadoghq.com/gus/windows-log-event/integrations/windows_event_log/

### CODE REVIEW

File: content/integrations/windows_event_log.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+ddtype: check
+---
+
+## Overview
+
+Collect Windows Events as logs in Datadog in order to:
+
+* Track system and application events in Datadog.
+* Correlate Windows system and application events with the rest of your application.
+* Track Security events and troubleshoot potential breach.
```
Comment (cswatt) [inline content/integrations/windows_event_log.md]: Should "Security" be capitalized? If it's the official name of a type of event, then yes, but if it's just general security-related events then it should be lowercase.

Also, for grammar, it should be "troubleshoot potential breaches."

File: content/integrations/windows_event_log.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Setup
+### Installation 
+The Windows event log check is included in the [Datadog Agent][4] package, so you don't need to install anything else on your server.
```
Comment (cswatt) [inline content/integrations/windows_event_log.md]: Capitalize "Windows Event" for consistency with the rest of the doc

**Comment by NBParis** [comment]
@l0k0ms I let you take the last review into account and merge it :)

**Comment by NBParis** [comment]
Thanks for that. Good for me

**Pull request merged**